mitiq/observable/tests/test_observable.py

```diff
@@ -32,6 +34,25 @@
 zmat = cirq.unitary(cirq.Z)
 
 
+# Executors.
+def execute(circuit: cirq.Circuit, shots: int = 8192) -> MeasurementResult:
+    result = cirq.Simulator().run(circuit, repetitions=shots)
+
+    n = sum(result._measurement_shape()[1].values())
+    bitstrings = [
+        list(map(int, np.binary_repr(value[0], width=n)))
+        for value in result.data.to_numpy()
+    ]
+    return MeasurementResult(
+        bitstrings,
+        qubit_indices=tuple(int(i) for i in result.data.columns[0].split(",")),
+    )
```

Comment:
Not important for this test but, just to sketch the typical workflow that  the user should apply, we could find something simpler.

For example, I find `result.measurements` more intuitive with respect to `result.data`, since the output is very similar to the input of `MeasurementResult`.

The following should work, even in the case of multiple measurements:

```suggestion
    return MeasurementResult(
        result=np.column_stack(list(result.measurements.values())),
        qubit_indices=tuple(
            int(q) for k in result.measurements.keys() for q in k.split(",")
        ),
    )
```

Reply:
Thanks! Much simpler, went with this.